docs/_build/authentication/requesttemplate.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Add authentication fields to Request Template
+order: 10
+layout: post-toc
+
+---
+
+# Add authentication fields to Request Template
+
+The Request Template is a request editor that lets users set static values that apply to all requests made by this integration. Users can configure the URL params, HTTP headers and request body. This is the perfect place to 
+set authentication fields.
+
+![](https://cdn.zappy.app/987abd7e65935ecb581fd356b45fb4bc.png)
+
+Adding fields to the Authentication Fields also writes the fields to the Request Template. 
+![](https://cdn.zappy.app/8cfafbc546e1693fa1e517ccc14a96ab.png)
+
+By adding fields to the Request Template, they will also have it show up in the Trigger and Actions request builder.
+The fields will be disabled with help text, and will prompt the user to go to the Request Template under Advanced Settings if they want to make changes to it.
```

Comment:
The applicable fields for authorization in triggers and actions under `Options` would then be disabled with help text. You'll need to access the Request Template under Advanced Settings to make changes to them.